deploy/examples/create-external-cluster-resources-tests.py

```diff
@@ -34,8 +34,7 @@
 class TestRadosJSON(unittest.TestCase):
     def setUp(self):
         print("\nI am in setup")
-        self.rjObj = ext.RadosJSON(['--rbd-data-pool-name=abc',
-                                    '--rgw-endpoint=10.10.212.122:9000', '--format=json'])
+        self.rjObj = ext.RadosJSON(['--rbd-data-pool-name=abc', '--format=json'])
```

Comment:
This is because we are now making the endpoint call through admin ops API, and there should be something that should be returned from the server, and the above endpoint is just random numbers
So as I mentioned it is not mockable for unit tests and it fails if we keep otherwise.